tests/test_field.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from dateutil import tz
+
+from elasticsearch_metrics.metrics import Date
```

Comment:
This kind of thing seems easy to overlook. There are still lots of examples of you inheriting from `Metric` instead of `metrics.Metric`, so if someone is just looking at the nearest code to copy, they might not notice that you want them to be importing fields from `elasticsearch_metrics.py`

Reply:
Good point. Better to make the usage consistent across the codebase--even in the tests. I'll change this to

```python
from elasticsearch_metrics import metrics

field = metrics.Date()
```